test/stake.test.js

```
@@ -0,0 +1,80 @@
+            it("withdraw reward", async function () {
+              await this.stake.connect(this.first).withdrawRewards()
+              await this.stake.connect(this.second).withdrawRewards()
+              await this.stake.connect(this.three).withdrawRewards()
+
+              await this.stake.withdrawUnusedRewards()
+              expect(await this.reward.balanceOf(this.first.address)).to.be.equal(7500)
+              expect(await this.reward.balanceOf(this.second.address)).to.be.equal(7500)
+              expect(await this.reward.balanceOf(this.three.address)).to.be.equal(5000)
+              expect(await this.reward.balanceOf(this.deployer.address)).to.be.equal(10000)
+            })
+          })
```

Biome (1.9.4)

[error] Excessive `describe()` nesting detected.

Excessive nesting of describe() calls can hinder test readability.
Consider refactoring and reduce the level of nested describe to improve code clarity.

(lint/complexity/noExcessiveNestedTestSuites)